### ISSUE TITLE
BCE loss during training

### ISSUE THREAD

**Issue by wangsen99** [tyhuang0428/SATNet#1]
Firstly, great job for MD.
However, when I understanding the training process, I notice that the loss function seems to be **cross_entropy** rather than **binary_cross_entropy** in your config satnet.py, which is differ from your paper.
Why don't you use BCE with logits and set 'num_classes' to 1 ?

Looking forward to your reply!

**Comment by tyhuang0428** [comment]
Sorry for the confusion. We do use cross_entropy here, and I will fix the claim in our paper. Thanks for your reminder!
Actually, we have conducted some experiments on the choices of the loss function. It seems that ce_loss and bce_loss have similar performances.

**Comment by wangsen99** [comment]
^_^
I see, good luck to your paper.
